packages/components/Molecules/Breadcrumbs/index.vue

```diff
@@ -34,6 +89,7 @@ export default {
   $rpl-breadcrumbs-background: rpl-color('white') !default;
   $rpl-breadcrumbs-border: 1px solid rpl-color('mid_neutral_1') !default;
   $rpl-breadcrumbs-border-radius: rem(4px) !default;
+  $rpl-breadcrumbs-padding-mobile: ($rpl-space-3) ($rpl-space * 4) !default;
```

Comment:
I chose not to add a -desktop suffix to the exisiting variable to avoid introducing a breaking change to any themes that are currently changing this value